b2share/modules/records/triggers.py

```diff
@@ -60,12 +70,18 @@ def set_record_schema(record, **kwargs):
     record['$schema'] = community_schema_json_schema_link(schema)
 
 
+def set_record_owner(record, **kwargs):
+    if not current_user.is_authenticated:
```

Comment:
Ok we can keep it like that for now. What about moving the owner-update code to the user input loader as suggested in my last comment? It would allow more flexibility for use cases such as the creation of already existing records by the admin (ex: when loading the data from B2Share 1.7 in 2.0).